code/modules/fish/fish_tank.dm

```diff
@@ -289,6 +292,18 @@
 					add_food(1)					//The corpse became food for the other fish, ecology at it's finest
 			if("glofish")
 				glo_light++
+			if("clownfish")
```

Comment:
This could be better if you add a `fish.special_interaction()` proc and you called it for each `fish` in `fish_list`.
A shame whomever PR'd fishtanks in the first place didn't thought of that. :^)

Reply:
Fish can't do that because they aren't objects, they are strings. No, really. We are not object-oriented at all here.

For instance, it would make sense to simply do fish.egg_type, right? No, we have to look up in a global associative list what the fish string translates to in terms of egg path because the fish is not an object, just a string.

Reply:
That's awful
After your PR is done, I'll refactor fish not to be strings but objects instead.